akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -460,7 +452,7 @@ private[akka] class Shard(
     }
   }
 
-  def deliverTo(id: EntityId, msg: Any, payload: Msg, snd: ActorRef): Unit = {
+  def deliverTo(id: EntityId, @unused msg: Any, payload: Msg, snd: ActorRef): Unit = {
```

Comment:
Since this is internal API, can we just remove that parameter entirely?

Reply:
It is used in an overload, so no.